pkg/controller/nginxingresscontroller/utils.go

```diff
@@ -115,7 +111,9 @@ func generatePodArgs(instance *k8sv1alpha1.NginxIngressController) []string {
 		}
 	}
 
-	if instance.Spec.EnableCRDs {
+	if instance.Spec.EnableCRDs != nil && !*instance.Spec.EnableCRDs {
+		args = append(args, "-enable-custom-resources=false")
```

Comment:
Just to confirm, the default is true, so if the instance.Spec.EnableCRDs != nil and instance.Spec.EnableCRDs is true, we don't need to append anything ?

Reply:
Yep. This is covered by: https://github.com/nginxinc/kubernetes-ingress/blob/b6b2807458697725eadaab087f8d323f8281a60b/cmd/nginx-ingress/main.go#L158